### ISSUE TITLE
Success Criterion 2.2.6 Accessible Authentication

### ISSUE THREAD

**Issue by GreggVan** [w3c/wcag21#354]
> Essential steps of an authentication process, which rely upon recalling or transcribing information, have one of the following:
> 
> alternative essential steps, which do not rely upon recalling and transcribing information; or
> an authentication-credentials reset process, which does not rely upon recalling and transcribing information
> Exceptions:
> 
> Authentication process involves basic identifying information to which the user has easy access, such as name, address, email address and identification or social security number.
> This is not achievable due to legal requirements.

==================================================================
It is not clear at all how this provision works.

As I read this -- it says 
"If there are essential steps that require "recalling or transcribing information"   you can pass if you have a RESET process that doesn't involve    "recalling or transcribing information".     
OK -- so I provide a simple RESET button that sends me new login and password.    But this doesn't change the authentication process so to authenticate after the reset still requires  "recalling or transcribing information".     So what is the benefit? 

Then there are the exceptions.    All of those things require memory.   Most are very common though so not hard to remember for most people except perhaps their social security number. Most people with cognitive disabilities I don't think know their social security number without  "recalling or transcribing" it.    But more to the point-- all but the SS # are public information so can't be used for any type of secure authentication.    And the SS# should not be used as a common authentication.

So it is not clear how this is of benefit or how it would work. 

Good intent but does not work unless you are talking about requiring biometric authentication? 

I tried clicking on UNDERSTANDING to figure it out -- but there was nothing there.   You really need to write the UNDERSTANDING documents before you release these.  OFTEN we found that we discovered all sorts of problems or faults in logic when trying to write the UNDERSTANDING document and EXAMPLES. 

Not ready for level AA or A

**Comment by JohnRochfordUMMS** [comment]
Hi Gregg,



I am the manager for the Accessible Authentication SC. I submitted an understanding section some time ago. Lisa Seeman told me today that Michael Cooper is working on publishing it within the next few days.



I hope it’s all you ever wanted, and more.

John

John Rochford<http://bit.ly/profile-rj>
UMass Medical School/E.K. Shriver Center
Director, INDEX Program
Instructor, Family Medicine & Community Health
www.DisabilityInfo.org
Twitter: @ClearHelper<https://twitter.com/clearhelper>

Confidentiality Notice:
This e-mail message, including any attachments, is for the sole use of the intended recipient(s) and may contain confidential, proprietary, and privileged information. Any unauthorized review, use, disclosure, or distribution is prohibited. If you are not the intended recipient, please contact the sender immediately and destroy or permanently delete all copies of the original message.


-----Original Message-----

**Comment by GreggVan** [comment]
Grin
thanks John
g

**Comment by JonathanPKatz** [comment]
For countries or locales with secure identity numbers, this criterion also does not work: those ID numbers tend to be difficult for many people to recall in any case.

**Comment by awkawk** [comment]
Official WG response: Thank you for your comment. This SC has been removed from the draft and will be able to be considered in a future revision.
